getting-started/windows-calico/quickstart.md

```diff
@@ -52,6 +55,42 @@ This is required to prevent Linux nodes from borrowing IP addresses from Windows
 calicoctl ipam configure --strictaffinity=true
 ```
 
+1. Prepare {{site.prodname}} networking
+
+   **VXLAN networking** 
+   
+   Disable BGP. 
+   ```bash
+   kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
+   ```
+   **BGP networking** 
+   
+   Enable BGP service on Windows node. 
+   Install the RemoteAccess service using the following Powershell commands:
+
+   ```powershell
+   PS C:\> Install-WindowsFeature RemoteAccess
+   PS C:\> Install-WindowsFeature RSAT-RemoteAccess-PowerShell
+   PS C:\> Install-WindowsFeature Routing
+   ```
+
+   Then restart the computer:
```

Comment:
@song-jiang could we restart after `Install-RemoteAccess -VpnType RoutingOnly` instead?

Reply:
No. It seems you have to restart computer before you can run `Install-RemoteAccess`.